test/e2e/docker/compose.yaml.tmpl

```diff
@@ -140,6 +140,22 @@ services:
         {{ if $.Network }}ipv4_address: 10.186.73.200{{ end }}
 {{end}}
 
+{{- if .Monitor }}
+  monitor:
+    labels:
+      e2e: true
+    container_name: monitor
+    image: omniops/monitor:{{or .MonitorTag "main"}}
```

Comment:
would `RelayerTag` and `MonitorTag` ever be different? Lets combine it into a single `.OmniTag`?

Reply:
I added this for debug purposes potentially, however, if we are debugging we can hard-code the tag. I'll use the same tag for now.